apps/fault/src/main.c

```diff
@@ -189,6 +189,7 @@ static void measure_vm_fault_handler_fn(int argc, char **argv)
     ccnt_t end;
     fault_results_t *results;
     seL4_Word badge = 0;
+    int msg = 0;
```

Comment:
why bring this out of the loop? It's not formatting actually, and the loop body scope for this seems all that is needed. The value does not have to survive the current iteration.